### PR TITLE
rpc: support stringified address in getnep5balances RPC

### LINKED ISSUE #1143: getnep5balances should accept addresses in parameters
At the moment it only accepts hex-encoded script hashes, but it should also work with regular address strings as it does work with C# module.

### PR DESCRIPTION
**Pull request by fyrchik**
Fix #1143 .

Also extend `getnep5transfers`.

### CODE REVIEW

File: pkg/rpc/server/server.go [hunk 1]
```diff
@@ -566,9 +566,13 @@ func (s *Server) getClaimable(ps request.Params) (interface{}, *response.Error)
 }
 
 func (s *Server) getNEP5Balances(ps request.Params) (interface{}, *response.Error) {
-	u, err := ps.ValueWithType(0, request.StringT).GetUint160FromHex()
+	val := ps.ValueWithType(0, request.StringT)
+	u, err := val.GetUint160FromHex()
```
Comment (roman-khimov) [inline pkg/rpc/server/server.go:570]: Maybe we need a `GetUint160FromAddresOrHex`?

Comment (fyrchik) [reply pkg/rpc/server/server.go:570]: Done.

**Comment by fyrchik** [comment]
@AnnaShaleva `GetNEP5Balances` works with raw hash so it is ok. Maybe we should update `GetNEP5Transfers` to work with raw `util.Uint160` type too.

**Comment by roman-khimov** [comment]
```
func (c *Client) GetNEP5Balances(address util.Uint160) (*result.NEP5Balances, error) 
func (c *Client) GetNEP5Transfers(address string) (*result.NEP5Transfers, error)
```
Seems to be OK already (although technically they could both be `util.Uint160`, but let's not change it for Neo 2.

**Pull request merged**